src/util/fp.js

```diff
@@ -30,6 +30,12 @@ export const mapKeys = curry((iteratee, obj) =>
   })
 );
 
+export const reverse = arr => arr.reverse();
+
+export const findLast = curry((predicate, arr) =>
```

Comment:
Uhm, why not use `findLast` from lodash/fp?

Reply:
I think this was here on your first commits. I'm not using it actually

Reply:
I thought you wanted to phase out lodash @felixjung ?

Reply:
We had a quick discussion about it the other day. I think it's healthier to use lodash actually. The lib is fully tested and has a big community behind it

Reply:
Sounds good - esp. with our need to support a few funny browsers. Thanks for the context 👍